server/registry/actions_apis_test.go

```diff
@@ -142,16 +156,6 @@ func TestCreateApiResponseCodes(t *testing.T) {
 			},
 			want: codes.NotFound,
 		},
-		{
-			desc: "missing resource body",
-			seed: &rpc.Project{Name: "projects/my-project"},
-			req: &rpc.CreateApiRequest{
-				Parent: "projects/my-project/locations/global",
-				ApiId:  "valid-id",
-				Api:    nil,
-			},
-			want: codes.InvalidArgument,
-		},
```

Comment:
would it be difficult to keep these tests and change the expected value to codes.OK?

Reply:
No, I actually did that first... but: 1. it's redundant with the other test and, 2: there are no other OK checks here.

Reply:
Does 1. mean there are other tests that pass nil resources to create command? I understand 2. but that's an artifact of the test organization and not really an indication of what we want to test.

Reply:
I added test for nil bodies that ensure that both there is no error and that the result is correct: https://github.com/apigee/registry/pull/1202/files#diff-25fea043835f743787d3ac3e0409c0e763af6e9bfdf07b0c7956394e3b604c48R78

Reply:
But it's fine. I'll add it back in if you'd like.

Reply:
Thanks, I thought that adding those back might have kept our coverage from dropping (very slightly, but that was surprising) - anyway, this LGTM. Thanks!

Reply:
I believe coverage percentage dropped because there is less code to cover. :)